pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -363,6 +374,33 @@ def replay_genargs_handle_argument(arg: str) -> str | None:
     return arg
 
 
+def get_cmake_compiler_flags() -> list[str]:
```

Comment:
Could you add a test for this to make codecov happy? If you are feeling generous, you could also add tests for `remove_avoided_requirements` and `install_reqs`.